cypress/support/oktaLogin.js

```diff
@@ -194,3 +200,98 @@ export const oktaLogin = (un, pw, storage_name, storage_email) => {
     })
   })
 }
```

Comment:
Sorry to keep asking for refactors here, but couldn't we just use the exiting oktaLogin? we pass in the user information that we want. We can get together and I can show you if you like.I don't think we need to repeat all this code.

Reply:
sure thing.. We can work together on this